lingua_franca/res/text/en-us/colors.json

```diff
@@ -0,0 +1,259 @@
+{
```

Comment:
Not sure I agree with all of the names here, i.e. hearing "Absolute Zero" would imply something close to black to me, but it's a shade of blue here

Reply:
all these come from list of colors wikipedia page

Reply:
https://en.wikipedia.org/wiki/List_of_colors:_A%E2%80%93F